PublicMapping/districtbuilder
# Add organization module to Import Project

## Overview

- Allows a user to associate an imported project with an organization by linking it with an organization template

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/132586006-79d98de3-c29c-4192-a92f-045259819d82.png)

![image](https://user-images.githubusercontent.com/66973361/132586016-d5a1a330-9ba9-453c-9600-6a9b96e3b3cb.png)


### Notes

There is no way for a user to associate a project with an organization besides through a template, right? It seems like we may need some additional error checking here to ensure that projects are not imported that violate the rules of a project template.

## Testing Instructions

- Create an organization and a project template for that organization, for a region that is not PA
- Join the newly created organization with your user
- Import Project screen -> Begin importing a project for the region associated with the project template
- Expect: Only newly created organization appears in options for selecting an organization
- Select the project template you created, and create the new project from the import
- Expect: Project now appears in all organization projects list

Closes #971
